### ISSUE TITLE
[Feature request] Show time remaining on map

### ISSUE THREAD

**Issue by asiatiki** [AHAAAAAAA/PokemonGo-Map#1677]
It would be great if we could see remaining time on top of the pokemon's icon so we don't have to click on each individual pokemon to get the information. Or maybe  offer the option in the sidebar?

**Comment by keaza** [comment]
https://pokevision.com/

like this?

**Comment by prajwalkman** [comment]
Pokevision's style obscures the pokemon sprites

**Comment by callmebackdraft** [comment]
@prajwalkman if it could be done with an option in nav everyone can choose,
i actually would like this very much ( dont know how to code it though )

**Comment by asiatiki** [comment]
Pokevision seems down, but something like this would be helpful

![screenshot_2016-07-23-20-04-41](https://cloud.githubusercontent.com/assets/18441536/17080931/db3428ec-5110-11e6-9d4e-210b77a3e348.png)

**Comment by codengine** [comment]
This would be really helpful. Looking foward to this.

**Comment by chuyskywalker** [comment]
Some 10min research suggests this https://www.npmjs.com/package/markerwithlabel may help.

**Comment by willgrnr** [comment]
I personally prefer to click to see the time. Seeing a bunch (read: a fucking lot) of timers everywhere gets cluttered real fast. 

Perhaps an option to toggle between timers on the sprites and timers in the popup would be best.

**Comment by callmebackdraft** [comment]
@willgrnr i agree with you there, was looking into this but have to update my current PR first

**Comment by devinjparsons** [comment]
Yes i think this would need to be a toggled option.
